sources/packages/web/src/composables/useRules.ts

```diff
@@ -31,9 +31,17 @@ export function useRules() {
     return "Expiry end date is required.";
   };
 
+  const checkStringNullOrEmptyRule = (value: string, fieldName: string) => {
+    if (value && value !== null && value.length > 0) {
```

Comment:
I believe this can be simplified as below:
```ts
return value?.length > 0 || `${fieldName} is required.`;
```

Reply:
renamed the method as discussed